### PR TITLE
[Geolocation] refactor: make point a property and fix function name

### PR DESCRIPTION
**Pull request by candux**
don't trigger useless renders.

I tried to split the functions but wasn't happy with it. Maybe one of you has a better idea

### CODE REVIEW

File: src/components/Geolocation/Geolocation.js [hunk 1]
```diff
@@ -145,10 +143,10 @@ class Geolocation extends PureComponent {
     this.setState({ active: true });
   }
 
-  highlight(point) {
+  highlight() {
     const { colorOrStyleFunc } = this.props;
     const feature = new Feature({
-      geometry: point,
+      geometry: this.point,
       source: new VectorSource(),
```
Comment (oterral) [inline src/components/Geolocation/Geolocation.js]: this line seems useless.

Comment (oterral) [inline src/components/Geolocation/Geolocation.js]:
```suggestion
```

Comment (candux) [reply src/components/Geolocation/Geolocation.js]: good catch!

**Comment by candux** [comment]
@oterral Good idea. I tried to do that

**Comment by mhaertwig** [comment]
@friedjoff can you check, please?

**Pull request merged**